tests/Api/Shop/OrdersTest.php

```diff
@@ -220,4 +221,54 @@ public function it_allows_to_patch_orders_payment_method(): void
 
         $this->assertResponse($response, 'shop/updated_payment_method_on_order_response', Response::HTTP_OK);
     }
+
+    /** @test */
+    public function it_allows_to_patch_orders_address(): void
+    {
+        $fixtures = $this->loadFixturesFromFiles(['authentication/customer.yaml', 'channel.yaml', 'cart.yaml', 'country.yaml', 'shipping_method.yaml', 'payment_method.yaml']);
```

Comment:
```suggestion
        $fixtures = $this->loadFixturesFromFiles(['channel.yaml', 'cart.yaml', 'country.yaml', 'shipping_method.yaml', 'payment_method.yaml']);
```
We may not load the user fixtures as the request is made as a guest